.github/workflows/CI.yml

```
@@ -36,3 +36,4 @@ jobs:
       - uses: codecov/codecov-action@v4
         with:
           file: lcov.info
+          token: ${{ secrets.CODECOV_TOKEN }}
```

By default all codecov errors are silent (ie do not fail CI), and hence these dependabots PRs were merged even though they broke codecov integration. Therefore in most repos I switched to
```suggestion
          token: ${{ secrets.CODECOV_TOKEN }}
          fail_ci_if_error: true
```